ios/Runner/Info.plist

```diff
@@ -53,7 +53,41 @@
 				<string>com.googleusercontent.apps.487707907739-084c8ksdv5gmha7m143rn01gl7r7tgum</string>
 			</array>
 		</dict>
+		<!-- End of the Google Sign-in Section -->
+		<!-- Facebook Sign-in Section-->
+		<!--
+		<dict>
+		... Some other CFBundleURLTypes definition.
+		</dict>
+		-->
+		<dict>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<!--
+				Replace "000000000000" with your Facebook App ID here.
```

Comment:
Could we clean out the instruction comments?  Personally I like to leave the comments that indicate the start and end of sections, like: `<!-- Facebook Sign-in Section-->` but get rid of the instructions. Just a preference obviously, feel free to ignore.